event/event.go

```diff
@@ -248,36 +248,41 @@ func (r *MessageSubscription) Serve(handler MessageHandler) error {
 				<-semaphore
 			}()
 
-			id, publishedTime := getMetadata(msg)
+			id, publishedTime := getMetadata(gocloudMsg)
+			msg := Message{
+				Body: gocloudMsg.Body,
+				Metadata: Metadata{
+					ID:            id,
+					PublishedTime: publishedTime,
+					Attributes:    gocloudMsg.Metadata,
```

Comment:
This is the confusing part...gocloud calls metadata what google calls attributes, pubsub has both...I ended up following the google design.